geom/src/polygon.rs

```diff
@@ -558,13 +559,20 @@ impl Polygon {
     }
 
     pub fn simplify(&self, epsilon: f64) -> Polygon {
-        to_geo(self.points()).simplifyvw_preserve(&epsilon).into()
+        self.to_geo().simplifyvw_preserve(&epsilon).into()
     }
 
     /// An arbitrary placeholder value, when Option types aren't worthwhile
     pub fn dummy() -> Self {
         Polygon::rectangle(0.1, 0.1)
     }
+
+    // A less verbose way of invoking the From/Into impl. Note this hides a potentially expensive
```

Comment:
Just calling this out -- my goal is to directly store a `geo::Polygon` here. Small steps!